library/styleguideComponents/Wrapper/styled.js

```diff
@@ -60,7 +60,15 @@ export const StyledContentText = styled.h2`
 `;
 
 export const StyledBadge = styled(Badge)`
```

Comment:
what we want is remove the Badge here and instead use something like:
```
export const StyledBadge = styled.span`
    // styles here
`
```
and then you can clean up all these styles and then use only the bold font
Also if you look at this file `src/styles/defaultTheme.js`
you can see the internal structure of the theme and access from the styled component to values like
```props.palette.text.main``` that already set the right text colors based on the main background